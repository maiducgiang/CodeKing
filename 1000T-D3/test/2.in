4
7587 6116 8771 4845
8208 7537 1699 8761
7257 2025 3086 8290
9758 7809 4025 8614
