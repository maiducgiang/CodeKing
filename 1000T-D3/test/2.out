42534161
26750755
28644252
70707175
